.github/CONTRIBUTING.md

```diff
@@ -98,6 +105,108 @@ Focused component needs to be clearly visible. This is handled in Stardust by fo
 
 In addition to basic focus handling, specific keyboard handlers can be added to the behaviors. These keyboard handlers call actions defined in the components, when corresponding keys are pressed by the user. For reference, see `keyActions` in [MenuItemBehavior][12] and `actionHandlers` in [MenuItem component][13].
 
+## Packages
+
+We are using [Lerna][14] to manage our packages and [Yarn Workspaces][15] to link them.
+
+### Add a new package
+
+#### Run `lerna create`
+ 
+You should to run `lerna create` command to create a new package
+
+- we are using `@stardust-ui` namespace on NPM to publish our packages
+- the directory name should not contain any namespace prefix and can be prefixed with the library name if the
+  implementation is not framework agnostic
+- please provide a meaningful description to a package in the matched field
+- use `https://github.com/stardust-ui/react/tree/master/packages/__DIRECTORY_NAME__` as `homepage`
+- our packages are published with MIT license, please follow it until you will have a specific requirements
```

Comment:
sorry, cannot completely understand what this part of the sentence about:

```
please follow it until you will have a specific requirements
```

Is it some general legal form? I mean, could we just mention something like that:

```
our packages are currently published with MIT license
```

Reply:
Updated to: `our packages are currently published with MIT license, please follow it until you will have specific legal requirements`